py3status/modules/glpi.py

```diff
@@ -13,6 +13,9 @@
     user: login user
     warning: set degraded color above this threshold
 
+Format placeholders:
+    {num_open} The number of open tickets
```

Comment:
Either of those are fine with me.  `{open}` seems too ambiguous without context.  `{tickets_open}` would be what I would lean towards.